### PR TITLE
Context adjustment

### LINKED ISSUE #38: Add controlled form state to question component
This will capture user input for each question
This will update each corresponding answer key on AnswerContext

### LINKED ISSUE #39: Add answerKey state to survey component
This answerKey will be in charge of storing all of the users answers
This information is what will ultimately be sent to the backend
This info will be sent to the question component with the updateState function to update accordingly

### LINKED ISSUE #40: User should be able to enter an answer for each question

### LINKED ISSUE #41: User should see next question once they submit the current question

### PR DESCRIPTION
**Pull request by BlakeDonn**
### What does this PR do?

* Adds dynamic rendering for the question being displayed to the user
* Adds logic to survey component for which question to render
* Adds local state to question
* Adds ability for question to alter survey state and answerContext

### Where should the reviewer start?

* Start by checking naming convention for altered files, ensure consistency

### How should this be manually tested?

* Get all prerequisite helpers running
* Input info into input field, press button and use dev tools to check state change

### Any background context you want to provide?

* It seems we will be able to conditionally render the question component pretty seamlessly with this logic
* We will need to implement some further logic to show other relevant question info (only question name currently displaying)

### What are the relevant tickets?

closes #38 
closes #39 
closes #40 
closes #41 

### Future notes:

* Add history capabilities to state so a user can move back questions
* Add logic to display current questions other relevant info

### CODE REVIEW

**Comment by BlakeDonn** [comment]
Fixed conflicts and commented out some logic to avoid styling issues

Should be easy to connect these pieces once we determine our routing / rendering direction

File: src/components/Survey/Survey.tsx [hunk 1]
```diff
@@ -5,26 +5,13 @@ import './Survey.css'
...
-    updateAnswers(questionKeys)
-    }
-  }, [thisContext]);
   
   return (
-    <AnswerContext.Provider value={answerKey}>
       <section className='survey-container'>
-
+        {/*<Question currentQuestion={currentQuestion} updateAllAnswers={updateAllAnswers}/> */}
         <div className='inner-container'>
```
Comment (timkeresey) [inline src/components/Survey/Survey.tsx:12]: updateAllAnswers is how we're updating context?

Comment (BlakeDonn) [reply src/components/Survey/Survey.tsx:12]: Yeah! this is a bit out of sorts as everything is commented out until we implement routing.

`updateAllAnswers` is what we are using to update AnswerContext, yep!

Currently I am working on some ideas for routing, and potentially moving AnswerContext up to App for the sake of simplicity

**Review by Atos20 — APPROVED** [review]
Can't wait to connect routing and context later tonight, thanks for the hard work on this.

**Pull request merged**